lib/maru/entity.ex

```diff
@@ -202,7 +206,7 @@ defmodule Maru.Entity do
     }
   end
 
-  defp do_parse(:if_func, %{options: options, runtime: runtime, information: information}) do
+  defp do_parse(:if_func, %{options: options, runtime: runtime, information: information}, caller) do
```

Comment:
for variable we not use int the function, add a prefix `_` is useful to fix the warning, change `caller` to `_caller` 🙂